jenkins/scripts/pulse_in_pipeline_scanning/utils/triage.py

```
@@ -0,0 +1,137 @@
+
+def call_triage_agent(risk_items: list) -> dict:
+    """POST risk items to the deployed triage agent; returns the parsed JSON response or {}."""
+    if not risk_items:
+        return {}
+    # The agent API expects input_message as a JSON string (same as query_agent.sh)
+    body = {"input_message": json.dumps(risk_items)}
+    try:
+        resp = requests.post(TRIAGE_AGENT_URL, json=body, timeout=TRIAGE_AGENT_TIMEOUT)
+        resp.raise_for_status()
+        return resp.json()
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)